### PR TITLE
Look for container state in containerd path

### PR DESCRIPTION
**Pull request by timstclair**
Fixes https://github.com/google/cadvisor/issues/1206

/cc @yujuhong

### CODE REVIEW

File: container/libcontainer/compatibility.go [hunk 1]
```diff
@@ -178,8 +178,12 @@ type v1State struct {
 	Config v1Config `json:"config"`
 }
 
-// Relative path to the libcontainer execdriver directory.
-const libcontainerExecDriverPath = "execdriver/native"
+const (
+	// Relative path to the libcontainer execdriver directory.
+	libcontainerExecDriverPath = "execdriver/native"
+	// Absolute path of the containerd directory.
+	containerdPath = "/run/containerd"
```
Comment (vishh) [inline container/libcontainer/compatibility.go]: For the future, we should see if we can get all the information we need via containerd APIs.

**Comment by vishh** [comment]
LGTM

File: container/libcontainer/compatibility.go (hunk 1 above)
Comment (yujuhong) [inline container/libcontainer/compatibility.go]: What information do we need from the config? Just curious. I can check the code of course :)

BTW, I think we can check the version of docker to decide where to look for the config, cant' we?

Comment (timstclair) [inline container/libcontainer/compatibility.go]: Yeah, I'll try to clean this up.

**Pull request merged**